docs/ops/sort/MulticlassNMS_8.md

```diff
@@ -0,0 +1,148 @@
+  * **Range of values**: `class`, `score`, `none`
+    * *class* - sort selected boxes by class id (ascending) in each batch element.
+    * *score* - sort selected boxes by score (descending) in each batch element.
+    * *none* - do not guarantee the order in each batch element.
+  * **Type**: `string`
+  * **Default value**: `none`
+  * **Required**: *No*
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: `i64` or `i32`
+  * **Type**: `string`
+  * **Default value**: `i32`
+  * **Required**: *No*
+
+* *iou_threshold*
+
+  * **Description**: intersection over union threshold.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+* *score_threshold*
+
+  * **Description**: minimum score to consider box for the processing.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+* *nms_top_k*
+
+  * **Description**: maximum number of boxes to be selected per class.
+  * **Range of values**: an integer starting from `-1`.
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *keep_top_k*
+
+  * **Description**: maximum number of boxes to be selected per batch element.
+  * **Range of values**: an integer starting from `-1`.
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer from `-1` to the maximum class id
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes
+  * **Required**: *No*
+
+* *nms_eta*
+
+  * **Description**: eta parameter for adpative NMS.
+  * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+  * **Type**: `float`
+  * **Default value**: `1.0`
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. he box cooridnates are layout as `[xmin, ymin, xmax, ymax]`. Required.
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. Required.
+
+**Outputs**:
+
+*   **1**: `selected_outputs` - tensor of type *T_THRESHOLDS* and shape `[number of selected boxes, 6]` containing the selected boxes with score and class as triplets `[class_id, box_score, xmin, ymin, xmax, ymax]`.
+
+*   **2**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, ]` the selected indices in the flattened `boxes`, which are absolute values cross batches.
+
+*   **3**: `selected_num` - 1D tensor of type *T_IND* and shape `[num_batches, ]` representing the number of selected boxes for each batch element.
+
+When there is no box selected, `selected_num` will be filled with 0. And both `selected_outputs` and `selected_indices` are null of shape (0,).
```

Comment:
"are null of shape (0,)." -> "are empty tensors with one of dimensions equal to `0`."